src/pb_stub.cc

```diff
@@ -478,15 +478,25 @@ Stub::Execute(RequestBatch* request_batch, ResponseBatch* response_batch)
 
   // Execute Response
   py::object execute_return = model_instance_.attr("execute")(request_list);
-  py::list responses;
+  py::object responses_obj;
   bool is_coroutine = asyncio.attr("iscoroutine")(execute_return).cast<bool>();
 
   if (is_coroutine) {
-    responses = asyncio.attr("run")(execute_return);
+    responses_obj = asyncio.attr("run")(execute_return);
   } else {
-    responses = execute_return;
+    responses_obj = execute_return;
   }
 
+  // Check the return type of execute function.
```

Comment:
Should add a test case in our CI for execute functions that don't return list.

Reply:
https://github.com/triton-inference-server/server/pull/3783